app/constants/urls.constants.ts

```diff
@@ -1,2 +1,5 @@
 export const checkUsername = (URL: string, username: string) =>
   `${URL}/users/usernameCheck/${username}`;
+
+export const getEvents = (HOST: string, startTime: number, endTime: number) =>
+  `${HOST}/events/calendar/${1}?startTime=${startTime}&endTime=${endTime}`;
```

Comment:
we should create `axios wrappers` here itself and just export the function

Reply:
@isVivek99  I do not agree. The purpose of this file is to server `url` constants. The axios wrapper should be in another file and we already have an issue raised for it.